internal/controller/nimservice_controller.go

```diff
@@ -256,6 +258,7 @@ func (r *NIMServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&rbacv1.RoleBinding{}).
 		Owns(&networkingv1.Ingress{}).
 		Owns(&autoscalingv2.HorizontalPodAutoscaler{}).
+		Owns(&kservev1beta1.InferenceService{}).
```

Comment:
We need to add CRD check here when platform is set to `kserve` similar to : https://github.com/NVIDIA/k8s-nim-operator/blob/main/internal/controller/nimservice_controller.go#L284

Reply:
Updated. Thanks.